### DISCUSSION TITLE
Menu not rotating through gaps

### DISCUSSION THREAD

**Discussion by lvedassp** [LanKuDot/game_modules#33] [Q&A] [answered]
I have a menu that requires a decent amount of gap to be available between two buttons. Whenever I try to rotate the menu, and if my cursor falls between the gaps between them. The menu does not rotate.
How do I fix this issue?

**Comment by LanKuDot** [comment, marked as answer]
Hi, the list uses the event system to detect the user input.
So you can create a ui image under it for detecting the user input.

1. Create an empty gameobject under the list, and drag all the boxes to it
2. Create an ui image under the list, and set it to be the first object in the hierarchy to make the boxes not be hidden by it.
3. Adjust this ui image to cover the dragging area you want, and set the color to be transparent.

The green area is the dragging area defined by the ui image created.
![圖片](https://user-images.githubusercontent.com/2039072/205073601-6192a08a-3be4-4d20-9d18-09b2aff2e70c.png)

And the hierarchy looks like this:
![圖片](https://user-images.githubusercontent.com/2039072/205073835-74a65655-f708-4c07-84e3-14aea33b0f37.png)
